tensorflow_recommenders/layers/factorized_top_k.py

```diff
@@ -656,7 +659,10 @@ def index(
         given, these will be used to as identifiers of top candidates returned
         when performing searches. If not given, indices into the candidates
         tensor will be given instead.
-
+      store_candidates: Optional boolean that controls whether passed candidate
```

Comment:
If the matrix is not necessary, let's just get rid of it unconditionally without adding this extra argument.

Reply:
Ok! I'll remove this option.